atom/browser/native_window_mac.mm

```diff
@@ -1289,6 +1305,21 @@ static bool FromV8(v8::Isolate* isolate,
   }
 }
 
+void NativeWindowMac::DeleteDelegate() {
+}
```

Comment:
When is this called (and do we need to do anything here)?

Reply:
It is not really needed, I'll remove it.